guide/webots-licenses.md

```diff
@@ -28,8 +28,8 @@ of charge and the users pay only for the modules they need. Various modules are
 available for purchase at different price tags. They include special robot
 models (e.g., NAO, DARwIn-OP, e-puck, Thymio 2, etc.) which can be programmed
 using different programming languages (e.g., C/C++, Matlab, Python) or
-interfaces to third parties software or hardware, etc. They are all described on
-Cyberbotics web site.
+interfaces to third party softwares or hardwares, etc. They are all described on the
```

Comment:
softwares &rarr; software
Also I believe hardware could be kept singular in this case.